react/components/Table/hooks/useOutsideCick.js

```diff
@@ -0,0 +1,13 @@
+import { useLayoutEffect } from 'react'
+
+const useOutsideClick = (ref, handlerFn, listenTrigger) => {
```

Comment:
Yep. I decided to do that on another PR. Exporting this envolve finding a logic to the table inherit states instead of calling a hook. (and remember, hooks cannot be called conditionally).

Reply:
I agree. But in that case, I guess that this should be in a dedicated `\hooks` folder